# 本文件仅用于雷达的网络配置
# 调用 network_setup 服务时，优先使用显示传参来配置雷达
# 当所有参数都是默认值时使用以下参数配置雷达(tab自动补齐即默认参数)

lslidar:
  lidar_ip: '192.168.1.200'       # 激光雷达自身IP地址
  destination_ip: '192.168.1.102' # 目的IP地址，数据接收端IP
  data_port: 2368                 # 目的数据端口
  dev_port: 2369                  # 目的设备端口